files/galaxy/tpv/destinations.yml.j2

```diff
@@ -244,12 +244,13 @@ destinations:
     inherits: pulsar_default
     runner: pulsar_eu_cz01
     max_accepted_cores: 32
-    max_accepted_mem: 128
+    max_accepted_mem: 256
     min_accepted_gpus: 0
     max_accepted_gpus: 0
     env:
       LC_ALL: C
-      SINGULARITY_CACHEDIR: "/storage/praha5-elixir/home/galaxyeu/singularity"
+      #SINGULARITY_CACHEDIR: "/storage/praha5-elixir/home/galaxyeu/singularity"
+      SINGULARITY_CACHEDIR: "/cvmfs/singularity.galaxyproject.org/all/"
```

Comment:
I don't think you can use the CVMFS as your Singularity cache directory. Singularity will usually download the images to this directory, and you would need write access to it. Please correct me if I am wrong.